esm/pretrained.py

```diff
@@ -74,6 +74,7 @@ def has_emb_layer_norm_before(model_state):
 
 
 def load_model_and_alphabet_core(model_data, regression_data=None):
+    import esm  # conditional esm.inverse_folding below
```

Comment:
```suggestion
```

I don't think this is necessary since esm is still imported at the top of the file?

Reply:
Yeah it is, I may miss something about import internals in python3, but without this it fails: the `import esm.X` statement below makes `esm` a local (undeclared) variable for the whole function scope

Reply:
Huh ok that's weird. Can't think of a better solution, so I think we can just merge this.